config/default.yml

```diff
@@ -298,8 +298,15 @@ RSpec/DescribedClassModuleWrapping:
 RSpec/Dialect:
   Description: Enforces custom RSpec dialects.
   Enabled: false
-  PreferredMethods: {}
+  PreferredMethods:
```

Comment:
It doesn’t feel right to change the config by default.
We can’t imply that those methods are always for capybara specs.
I can’t quickly find it, but I recall someone mentioned that ‘describe’ isn’t a safe correction for ‘feature’ that has a certain metadata attached to it by default.

I suggest instead adding what needs to be done to this message below:
```
in .rubocop.yml file, enable `RSpec/Dialect`.
```

Reply:
I agree. One thing we could do it to link from the obsoletion.yml message to https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecdialect, and add to the documentation in https://github.com/rubocop/rubocop-rspec/blob/6f340256dd1502cf23f20f6c6b7410e05de0c923/lib/rubocop/cop/rspec/dialect.rb#L42-L47


```diff
diff --git a/lib/rubocop/cop/rspec/dialect.rb b/lib/rubocop/cop/rspec/dialect.rb
index 7140541c..95d75161 100644
--- a/lib/rubocop/cop/rspec/dialect.rb
+++ b/lib/rubocop/cop/rspec/dialect.rb
@@ -42,6 +42,19 @@ module RuboCop
       #     # ...
       #   end
       #
+      # If you were previously using the `RSpec/Capybara/FeatureMethods` cop and
+      # want to keep disabling all Capybara-specific methods that have the same
+      # native RSpec method (e.g. are just aliases), use the following config:
+      #
+      #   RSpec/Dialect:
+      #     PreferredMethods:
+      #       background: :before
+      #       scenario:   :it
+      #       xscenario:  :xit
+      #       given:      :let
+      #       given!:     :let!
+      #       feature:    :describe
+      #
       class Dialect < Base
         extend AutoCorrector
         include MethodPreference
```